libs/cdk/src/lib/utils/components/dynamic-portal/dynamic-portal.component.ts

```diff
@@ -0,0 +1,49 @@
+import {
+    Component,
+    ElementRef,
+    inject,
+    Input,
+    OnChanges,
+    SimpleChanges,
+    TemplateRef,
+    ViewContainerRef
+} from '@angular/core';
+import { ComponentPortal, ComponentType, DomPortal, Portal, PortalModule, TemplatePortal } from '@angular/cdk/portal';
+import { coerceElement } from '@angular/cdk/coercion';
+
+@Component({
+    selector: 'fdk-dynamic-portal',
+    standalone: true,
+    imports: [PortalModule],
+    template: ` <ng-template [cdkPortalOutlet]="portal"></ng-template>`
+})
```

Comment:
Could we get a comment here explaining the purpose of this component? It's unclear just looking at the file